test/web-platform-tests/to-run.yaml

```
@@ -163,7 +164,9 @@ document-level-touchmove-event-listener-passive-by-default.html: [timeout, Uses
 event-global-extra.window.html: [timeout, Not implemented]
 event-global.html: [fail, Not implemented]
 relatedTarget.window.html: [fail, Unknown]
-scrolling/**: [fail, Scroll methods not implemented]
+scrolling/input-text-scroll-event-when-using-arrow-keys.html: [timeout, Fires UnhandledPromiseRejectionWarning]
+scrolling/overscroll-**.html: [fail, Scroll methods not implemented]
+scrolling/scrollend-**.html: [fail, Scroll methods not implemented]
```

This should probably work with:
```suggestion
scrolling/**: [fail, Scroll methods not implemented]
```

since `%Array.prototype.find%` returns the first matching value.